src/sampling.jl

```diff
@@ -720,18 +886,18 @@ function efraimidis_aexpj_wsample_norep!(rng::AbstractRNG, a::AbstractArray,
 
     # set threshold
     @inbounds threshold = pq[1].first
-    X = threshold*randexp(rng)
+    X = threshold * randexp(rng)
 
-    @inbounds for i in s+1:n
+    @inbounds for i = s+1:n
         w = wv.values[i]
         w < 0 && error("Negative weight found in weight vector at index $i")
         w > 0 || continue
         X -= w
         X <= 0 || continue
 
         # update priority queue
-        t = exp(-w/threshold)
-        pq[1] = (-w/log(t+rand(rng)*(1-t)) => i)
+        t = exp(-w / threshold)
+        pq[1] = (-w / log(t + rand(rng) * (1 - t)) => i)
```

Comment:
```suggestion
        pq[1] = (-w/log(t + rand(rng)*(1 - t)) => i)
```
Use spacing to indicate order of operations, no need to put spaces around high precedence operators. Applies elsewhere.

Reply:
I had the Format Code option in Juno IDE. I think that caused these changes. I'll fix this everywhere.